content/en/opentelemetry/interoperability/environment_variable_support.md

```diff
@@ -0,0 +1,155 @@
+
+[for more info, see the OpenTelemetry documentation][9]
+## General SDK Configuration
+
+`OTEL_SERVICE_NAME`
+: **Corresponding Datadog Environment Variable**: `DD_SERVICE`<br>
+Sets the service name<br>
+**Notes**: If `service.name` is also provided in `OTEL_RESOURCE_ATTRIBUTES`, then `OTEL_SERVICE_NAME` takes precedence<br>
+
+`OTEL_LOG_LEVEL`
+: **Corresponding Datadog Environment Variable**: `DD_LOG_LEVEL`<br>
+Log level used by the SDK logger<br>
+**Notes**: A log level of debug will also map to `DD_TRACE_DEBUG=true`<br>
+This maps to `DD_TRACE_LOG_LEVEL` in `nodejs` & `php` sdk's<br>
+**Not Supported In**: `python`, `dotnet`, `ruby`, & `golang` sdk's<br>
+
+`OTEL_PROPAGATORS`
+: **Corresponding Datadog Environment Variable**: `DD_TRACE_PROPAGATION_STYLE`<br>
+Propagators to be used as a comma-separated list<br>
+**Notes**: the only supported values are `tracecontext`, `b3`, `b3multi`, `none`, `datadog` along with `xray` for the `java` sdk<br>
+Values MUST be deduplicated in order to register a `Propagator` only once<br>
+
+`OTEL_TRACES_SAMPLER & OTEL_TRACES_SAMPLER_ARG`
+: **Corresponding Datadog Environment Variable**: `DD_TRACE_SAMPLE_RATE`<br>
+`OTEL_TRACES_SAMPLER`: Sampler to be used for traces & `OTEL_TRACES_SAMPLER_ARG`: String value to be used as the sampler argument<br>
+**Notes**: The specified value will only be used if `OTEL_TRACES_SAMPLER` is set. Each Sampler type defines its own expected input, if any. Invalid or unrecognized input MUST be logged and MUST be otherwise ignored, i.e. the implementation MUST behave as if `OTEL_TRACES_SAMPLER_ARG` is not set<br>
+Mapped values between `OTEL_TRACES_SAMPLER` & `DD_TRACE_SAMPLE_RATE`:<br>
+  - `parentbased_always_on`|`1.0`
+  - `parentbased_always_off`|`0.0`
+  - `parentbased_traceidratio`|`${OTEL_TRACES_SAMPLER_ARG}`
+  - `always_on`|`1.0`
+  - `always_off`|`0.0`
+  - `traceidratio`|`${OTEL_TRACES_SAMPLER_ARG}`
+
+`OTEL_TRACES_EXPORTER`
+: **Corresponding Datadog Environment Variable**: `DD_TRACE_ENABLED=false` <br>
+Trace exporter to be used<br>
+**Notes**: only a value of `none ` is accepted<br>
+
+`OTEL_METRICS_EXPORTER`
+: **Corresponding Datadog Environment Variable**: `DD_RUNTIME_METRICS_ENABLED=false` <br>
+Metrics exporter to be used<br>
+**Notes**: only a value of `none` is accepted<br>
+**Not Supported In**: `php` sdk<br>
+
+`OTEL_RESOURCE_ATTRIBUTES`
+: **Corresponding Datadog Environment Variable**: `DD_TAGS` <br>
+Key-value pairs to be used as resource attributes. See [Resource semantic conventions][11] for details<br>
+**Notes**: `deployment.environment` maps to the `DD_ENV` environment variable<br>
+`service.name` maps to the `DD_SERVICE` environment variable<br>
+`service.version` maps to the `DD_VERSION` environment variable
```

Comment:
We should note that only the first 10 will be picked up, subsequent values dropped